### ISSUE TITLE
导出微调模型在c#中使用PaddleOcrAll()方法报错，官方推理模型正常加载

### ISSUE THREAD

**Issue by gunh4mmer** [PaddlePaddle/PaddleOCR#15950]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

使用[https://paddlepaddle.github.io/PaddleOCR/latest/version3.x/module_usage/text_detection.html#_2](url)中的ppocrv5的server训练模型进行模型微调，训练完毕之后使用官方export_model.py导出静态图模型，在c#中进行加载（最小代码见下方），报错外部组件发生异常，但是使用上方网址中提供的推理模型正常加载并推理成果；训练config有部分修改，见下方：
Global:
  model_name: PP-OCRv5_server_det # To use static model for inference.
  debug: false
  use_gpu: true
  epoch_num: &epoch_num 500
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/PP-OCRv5_server_det
  save_epoch_step: 10
  eval_batch_step:
  - 0
  - 1500
  cal_metric_during_train: false
  checkpoints: 
  pretrained_model: /home/zzj/paddleocrv5/PaddleOCR/pretrained/PP-OCRv5_server_det_pretrained.pdparams
  save_inference_dir: null
  use_visualdl: false
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./checkpoints/det_db/predicts_db.txt
  distributed: true

Architecture:
  model_type: det
  algorithm: DB
  Transform: null
  Backbone:
    name: PPHGNetV2_B4
    det: True
  Neck:
    name: LKPAN
    out_channels: 256
    intracl: true
  Head:
    name: PFHeadLocal
    k: 50
    mode: "large"
    

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001 #(8*8c)
    warmup_epoch: 2
  regularizer:
    name: L2
    factor: 1e-6

PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 2.0

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: /home/zzj/paddleocrv5/train_data/det
    label_file_list:
      - /home/zzj/paddleocrv5/train_data/det/train.txt
    ratio_list: [1.0]
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - CopyPaste: null
    - IaaAugment:
        augmenter_args:
        - type: Fliplr
          args:
            p: 0.5
        - type: Affine
          args:
            rotate:
            - -10
            - 10
        - type: Resize
          args:
            size:
            - 0.5
            - 3
    - EastRandomCropData:
        size:
        - 640
        - 640
        max_tries: 50
        keep_ratio: true
    - MakeBorderMap:
        shrink_ratio: 0.4
        thresh_min: 0.3
        thresh_max: 0.7
        total_epoch: *epoch_num
    - MakeShrinkMap:
        shrink_ratio: 0.4
        min_text_size: 8
        total_epoch: *epoch_num
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - threshold_map
        - threshold_mask
        - shrink_map
        - shrink_mask
  loader:
    shuffle: true
    drop_last: false
    batch_size_per_card: 8
    num_workers: 16

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: /home/zzj/paddleocrv5/train_data/det
    label_file_list:
      - /home/zzj/paddleocrv5/train_data/det/val.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - DetResizeForTest:
        limit_side_len: 960
        limit_type: max
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - shape
        - polys
        - ignore_tags
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 1
    num_workers: 8
profiler_options: null

### 🏃‍♂️ Environment (运行环境)

训练环境：
cuda11.8 
cudnn 8.9
paddlepaddle 3.1
paddleocr 3.1

部署环境：
cpu环境I7-12700H
sdcb.paddleOCR 3.0.1
sdcb.inference 3.0.1
Sdcb.PaddleInference.runtime.win64.mkl 3.0.0.51

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

DetectionModel test1 = Sdcb.PaddleOCR.Models.Local.LocalDetectionModel.FromDirectory("D:\\desktop\\PP-OCRv5_server_det_infer-自己", ModelVersion.V5);
ClassificationModel test2 = Sdcb.PaddleOCR.Models.Local.LocalClassificationModel.FromDirectory("D:\\desktop\\PP-LCNet_x1_0_textline_ori_infer",ModelVersion.V5);
RecognizationModel test3 = Sdcb.PaddleOCR.Models.Local.LocalRecognizationModel.FromDirectory("D:\\desktop\\PP-OCRv5_server_rec_infer", "E:\\path_to_labelme\\PaddleOCR-release-2.10\\ppocr\\utils\\dict\\chinese_cht_dict.txt", ModelVersion.V5);
FullOcrModel model = new FullOcrModel ( test1, test2, test3 );
byte[] sampleImageData;
using (PaddleOcrAll all = new PaddleOcrAll(model, PaddleDevice.Mkldnn())
{
    AllowRotateDetection = true, /* 允许识别有角度的文字 */
    Enable180Classification = true, /* 允许识别旋转角度大于90度的文字 */
})
